src/ansys/dpf/core/field.py

```diff
@@ -804,6 +806,27 @@ def meshed_region(self) -> MeshedRegion:
     def meshed_region(self, value: MeshedRegion):
         self._set_support(support=value, support_type="MESHED_REGION")
 
+    @property
+    def header(self) -> DataTree:
+        """Field Header, which stores metadata of the Field.
+
+        Returns
+        -------
+        :class:`ansys.dpf.core.data_tree.DataTree`
+
+        """
+        server_meet_version_and_raise(
```

Comment:
@rafacanton you can use `@version_requires` decorator instead, see [here for example](https://github.com/ansys/pydpf-core/blob/1095ea27673bcd5a51d1388d21bb3a88a071d6ba/src/ansys/dpf/core/collection_base.py#L125)